crm_lead_currency/views/crm_lead_views.xml

```diff
@@ -12,15 +12,10 @@
             >
                 <field name="customer_currency_id" />
                 <field name="is_same_currency" invisible="1" />
-                <field
-                    name="amount_customer_currency"
-                    attrs="{'invisible': [('is_same_currency', '=', True)]}"
-                />
+                <field name="amount_customer_currency" invisible="is_same_currency" />
             </xpath>
             <field name="expected_revenue" position="attributes">
-                <attribute name="attrs">
-                    {'readonly': [('is_same_currency', '=', False)]}
-                </attribute>
+                <attribute name="readonly" add="(not is_same_currency)" />
```

Comment:
<attribute name="readonly">not is_same_currency</attribute>

Reply:
Done, ready to review